helm/grafana/values.yaml

```diff
@@ -5,6 +5,19 @@ global:
 ciliumNetworkPolicy:
   enabled: true
 
+## Grafana postgresql database
+database:
+  enabled: false
+  # The postgresql cluster name will be <release-name>-<clusterName>
+  clusterName: postgresql
+  # rovider is either "capa" or "capz"
+  provider: ""
```

Comment:
I don't think we need provider and baseurl without hte backup

Reply:
We don't need provider but we need baseUrl for the bucket.

Reply:
But the bucket is needed for the backups :)

Reply:
Oh yeah you're right !